swift/swiftlint/SwiftLint-Package.swift

```diff
@@ -0,0 +1,120 @@
+// swift-tools-version:5.7
```

Comment:
the linter is failing due to missing license header:
```
 apache-rat license violation: swift/swiftlint/SwiftLint-Package.swift
```
https://github.com/apache/arrow/actions/runs/6590875064/job/17908328499?pr=38133#step:5:1052

Reply:
Good catch, thank you!  Will add.